reporter/src/main/kotlin/StatisticsCalculator.kt

```diff
@@ -158,3 +172,6 @@ internal object StatisticsCalculator {
         )
     }
 }
+
+private fun secondsBetween(startInclusive: Instant, endInclusive: Instant) =
+    java.time.Duration.between(startInclusive, endInclusive).toKotlinDuration().inWholeSeconds
```

Comment:
What is the reason to use the fully qualified name instead of an import for `java.time.Duration`?

Reply:
I'm using `Kotlin.time.Duration` as well, so the imports would clash.

Reply:
You're never directly using the type `kotlin.time.Duration`. Only indirectly through `.toKotlinDuration()`.
There is no import for `kotlin.time.Duration`.

Reply:
Oh, right. Fixed it.